packages/checkbox/checkbox.test.mts

```diff
@@ -134,6 +136,7 @@ describe('checkbox prompt', () => {
     `);
 
     events.keypress('enter');
+    await Promise.resolve();
     expect(getScreen()).toMatchInlineSnapshot('"? Select a number 1"');
 
     await expect(answer).resolves.toEqual([1]);
```

Comment:
```suggestion
    events.keypress('enter');
    await expect(answer).resolves.toEqual([1]);
    expect(getScreen()).toMatchInlineSnapshot('"? Select a number 1"');
```

Let's flip those so we await the answer, and then check the output.

I'm not against using `await Promise.resolve()` if strictly necessary; but those assertions appear to just be in the wrong order.